system/settings/settings.xml

```diff
@@ -2249,7 +2253,7 @@
         </setting>
         <setting id="audiooutput.ac3transcode" type="boolean" parent="audiooutput.ac3passthrough" label="667" help="36429">
           <level>2</level>
-          <default>false</default>
+          <default>true</default>
```

Comment:
That was the one thing I wasn't sure on, is what performance impact if any would be on RPi or any other low power platforms, other than that couldn't see any issues in my testing.